package.json

```diff
@@ -49,7 +50,14 @@
     "debug:build:prod": "node-nightly --inspect --debug-brk node_modules/webpack/bin/webpack.js  --config webpack.prod.config.ts",
     "docs": "typedoc --options typedoc.json ./src",
     "lint": "tslint \"src/**/*.ts\" || true",
-    "global": "npm install -g angular-cli nodemon npm-check-updates rimraf ts-node typedoc typescript webpack webpack-bundle-size-analyzer rollup marked node-gyp"
+    "global": "npm install -g angular-cli nodemon npm-check-updates rimraf ts-node typedoc typescript webpack webpack-bundle-size-analyzer rollup marked node-gyp protractor",
+    "postglobal": "npm link protractor",
```

Comment:
In that case I think the solution should be to add protractor as a local, as well as a global dependency, just like  currently e.g. [webpack is installed as a global dependency](https://github.com/DSpace/dspace-angular/blob/6f3229fb3858847a3ceb6eda38d4d206b8b3a92a/package.json#L52), to be able to use it from the command-line, but [also added as a dependency of the project](https://github.com/DSpace/dspace-angular/blob/6f3229fb3858847a3ceb6eda38d4d206b8b3a92a/package.json#L133) so it can be used by other libs.

[`npm link package-name` is designed to be used during development of an npm package](https://60devs.com/simple-way-to-manage-local-node-module-using-npm-link.html), to be able to use it as a library in a second project it before it's hosted on npm